### PR TITLE
autorelease brew

### PR DESCRIPTION
**Pull request by egze**
The plan:

1. Clone `homebrew-brewed` to a tmp folder.
2. Replace file hashes and version in `phrase.rb`.
3. push to master

### CODE REVIEW

File: clients/cli/build/update_brew.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Change the sha256 in Formula/phrase.rb
+for file in ${FILES_FOR_BREW[@]}; do
+  sha256=$(sha256sum "$DIST_DIR/$file" | awk '{ print $1 }')
+  sed -i '' "s/sha256 \".*\" # $file/sha256 \"$sha256\" # $file/g" "$brew_dir/Formula/phrase.rb"
+done
+
+# Change the version in Formula/phrase.rb
+sed -i '' "s/version \".*\"/version \"$VERSION\"/g" "$brew_dir/Formula/phrase.rb"
+
+current=$(pwd)
```
Comment (tobstarr) [inline clients/cli/build/update_brew.sh:20]: you could also use `pushd ${pwd}` and then `popd` when done

File: clients/cli/build/update_brew.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo pipefail
+
+brew_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+git clone --depth 1 https://$API_TOKEN_GITHUB@github.com/phrase/homebrew-brewed.git $brew_dir &> /dev/null
+
+DIST_DIR="./dist"
+FILES_FOR_BREW=("phrase_linux_386.tar.gz" "phrase_linux_amd64.tar.gz" "phrase_macosx_amd64.tar.gz")
```
Comment (tobstarr) [inline clients/cli/build/update_brew.sh:9]: I would have probably just used 
`for file in phrase_linux_386.tar.gz phrase_linux_amd64.tar.gz phrase_macosx_amd64.tar.gz; do`
or maybe even `ls ${DIST_DIR}/phrase_*.tar.gz`? Only a matter of time when we also need `phrase_macosx_arm.tar.gz`.

File: clients/cli/build/update_brew.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eo pipefail
+
+brew_dir=$(mktemp -d -t ci-XXXXXXXXXX)
+git clone --depth 1 https://$API_TOKEN_GITHUB@github.com/phrase/homebrew-brewed.git $brew_dir &> /dev/null
```
Comment (tobstarr) [inline clients/cli/build/update_brew.sh:6]: I would probably now swallow all output here as there could be plenty of reasons why the cloning could not work.

Maybe write the output some where tmp and then cat if the command fails?

**Review by tobstarr — APPROVED** [review]

File: clients/cli/build/update_brew.sh (hunk 1 above)
Comment (egze) [reply clients/cli/build/update_brew.sh:24]: good to know. will use it next time 👍

File: clients/cli/build/update_brew.sh (hunk 2 above)
Comment (egze) [reply clients/cli/build/update_brew.sh:13]: Not all files are needed, for example windows is skipped.

**Review by tobstarr — APPROVED** [review]

**Pull request merged**